README.md

```diff
@@ -22,7 +22,7 @@ console.log(results);
 ```
 
 ```js
-// Your answer...
+Because you are using find instead of findOne, the results will be an array of objects.
 ```
```

Comment:
The issue is that `.find` is asynchronous, meaning that you need to `console.log(results)` inside of a callback or promise.